miio/integrations/yeelight/specs.yaml

```diff
@@ -1,180 +1,200 @@
 yeelink.light.bslamp1:
-  color_temp: [1700, 6500]
-  night_light: False
-  background_light: False
-  supports_color: True
+  main:
+    color_temp: [1700, 6500]
+    night_light: False
+    supports_color: True
 yeelink.light.bslamp2:
-  color_temp: [1700, 6500]
-  night_light: True
-  background_light: False
-  supports_color: True
+  main:
+    color_temp: [1700, 6500]
+    night_light: True
+    supports_color: True
 yeelink.light.bslamp3:
-  color_temp: [1700, 6500]
-  night_light: True
-  background_light: False
-  supports_color: True
+  main:
+    color_temp: [1700, 6500]
+    night_light: True
+    supports_color: True
 yeelink.light.ceil26:
-  color_temp: [2700, 6500]
-  night_light: True
-  background_light: False
-  supports_color: False
+  main:
+    color_temp: [2700, 6500]
+    night_light: True
+    supports_color: False
 yeelink.light.ceila:
-  color_temp: [2700, 6500]
-  night_light: True
-  background_light: False
-  supports_color: False
+  main:
+    color_temp: [2700, 6500]
+    night_light: True
+    supports_color: False
 yeelink.light.ceiling1:
-  color_temp: [2700, 6500]
-  night_light: True
-  background_light: False
-  supports_color: False
+  main:
+    color_temp: [2700, 6500]
+    night_light: True
+    supports_color: False
 yeelink.light.ceiling2:
-  color_temp: [2700, 6500]
-  night_light: True
-  background_light: False
-  supports_color: False
+  main:
+    color_temp: [2700, 6500]
+    night_light: True
+    supports_color: False
 yeelink.light.ceiling3:
-  color_temp: [2700, 6500]
-  night_light: True
-  background_light: False
-  supports_color: False
+  main:
+    color_temp: [2700, 6500]
+    night_light: True
+    supports_color: False
 yeelink.light.ceiling4:
-  color_temp: [2700, 6500]
-  night_light: True
-  background_light: True
-  supports_color: False
+  main:
+    color_temp: [2700, 6500]
+    night_light: True
+    supports_color: False
+  background:
```

Comment:
Okay, I took another look into this. Please let move the main light to be on the main level in the specs as that makes much more sense like that API wise and avoids one level of unnecessary nesting.

On the topic of background & night light, are any features of the night light controllable? That is, do we need to have color temp, supports color or any other information for them? If yes, maybe it will make sense to have a `sublights` list of available lights (excluding the main one).

Reply:
To avoid missunderstanding night light the same as moonlight. (I don`t know why but is moonlight in the app and night light in the documentation) Now I think that use it as separate light could work. But as HA user I dont wont to have two mutually exclusive entities, because moonlight or night light use one physic lite as main light.

For background  - this is separate led light.

![image](https://user-images.githubusercontent.com/4723134/140105340-00dea1ef-90a9-4584-857f-96992c5f9bdb.png)
Moonlight/Night light (color temp is not available in this mode)
![image](https://user-images.githubusercontent.com/4723134/140105412-8f859792-7804-4ea8-bf21-9de0346dd77e.png)
Main Light
![image](https://user-images.githubusercontent.com/4723134/140105454-5d5fbd23-3362-41c0-acaf-ca3c04eae6a0.png)
background / Ambient light (in app and doc different name too)
![image](https://user-images.githubusercontent.com/4723134/140105512-02a52455-9fd3-4a11-8183-712f5d43e0aa.png)
In the MIHome we have 1 devise with 2 lights and moon light is a mode for main. Background looks like color bulb interfase (if you whant can add screen for Background ). 

@rytilahti please confirm that there was no misunderstanding? 

PS: My SSD is dead. It took me a couple of days to reinstall and set up the whole environment on a new one.

Reply:
Hi @Kirmas , I hope you got your data back after the SSD failure!

So, I agree that there should be only two lights while the moon light / night mode should be a toggle. IMO the yaml should always define the main light, so that's why I think it should be on the top level in the spec file instead of being hidden under the "main".

Reply:
> Hi @Kirmas , I hope you got your data back after the SSD failure!

Backups - our all.

> So, I agree that there should be only two lights while the moon light / night mode should be a toggle. IMO the yaml should always define the main light, so that's why I think it should be on the top level in the spec file instead of being hidden under the "main".

Is better after https://github.com/rytilahti/python-miio/pull/1166/commits/6a9bcb15681ec638c1efda9b2e7ced7109588085?

Reply:
Looks better, let's get this merged and fix whatever comes up in the upcoming PRs. Thanks for your patience again :-)